src/Compilers/Core/Portable/Operations/ILocalFunctionOperation.cs

```diff
@@ -23,10 +23,18 @@ public interface ILocalFunctionOperation : IOperation
         /// Body of the local function.
         /// </summary>
         /// <remarks>
-        /// Expression-bodied local functions are normalized here to be a single <see cref="IOperation"/>
-        /// inside of a block statement.
+        /// This will return the <see cref="BlockBody"/> if it exists, and the <see cref="ExpressionBody"/> if the <see cref="BlockBody"/> does not exist.
+        /// If both exist, this will return just the <see cref="BlockBody"/>, and you must use <see cref="ExpressionBody"/> to retrieve the expression body.
         /// </remarks>
         IBlockOperation Body { get; }
+        /// <summary>
+        /// The block body of the local function, if it exists.
+        /// </summary>
+        IBlockOperation BlockBody { get; }
```

Comment:
>IBlockOperation BlockBody { get; } [](start = 8, length = 34)

I think we should simply add one property```IBlockOperation IgnoredBody``` , similar to IWhileLoopOperation.

Reply:
Design Decision: Do this.